packages/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/android/AndroidLibraryExtension.kt

```diff
@@ -0,0 +1,23 @@
+package expo.modules.plugin.android
+
+import com.android.build.gradle.LibraryExtension
+
+internal fun LibraryExtension.applySDKVersions(compileSdk: Int, minSdk: Int, targetSdk: Int) {
+  this.compileSdk = compileSdk
+  defaultConfig {
+    this@defaultConfig.minSdk = minSdk
+    this@defaultConfig.targetSdk = targetSdk
```

Comment:
not related. yesterday i learnt that `defaultConfig.targetSdk` is deprecated with `lint.targetSdk` or `test.targetSdk`: https://android.googlesource.com/platform/tools/base/+/refs/heads/mirror-goog-studio-main/build-system/gradle-api/src/main/java/com/android/build/api/dsl/LibraryBaseFlavor.kt#28

Reply:
Ok, we can remove it. I'll handle that in a separate PR